scripts/install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+os=`echo "$(uname -s)_$(uname -m)"`
+if [ $os == "Linux_x86_64" ] || [ $os == "Linux_armv6" ] || [ $os == "Darwin_x86_64" ] || [ $os == "Linux_arm64" ] || [ $os == "Linux_i386" ]]
+then
+        echo Installing Phoneinfoga.........
+        wget "https://github.com/sundowndev/phoneinfoga/releases/download/v2.3.2/phoneinfoga_$(uname -s)_$(uname -m).tar.gz"
```

Comment:
Would be cool to fetch the latest tag automatically.

An example here : https://github.com/sundowndev/go-covermyass#installation